### PR TITLE
Font Awesome update & invisible icon fix

### PR DESCRIPTION
**Pull request by mzgnr**

### CODE REVIEW

File: stylesheets/main.scss [hunk 1]
```diff
@@ -387,3 +387,7 @@ pre {
 .fixedsticky {
   top: 0;
 }
+
+.fa {
+  transform: translate(0, 0);
+}
```
Comment (alebcay) [inline stylesheets/main.scss]: You might need a newline here.

**Comment by joshtoo** [comment]
For future reference. You create branches off of the branch you intend to open a pull request to. 

Master will be a protected branch and impossible to commit to. There will be a diagram on the wiki in an hour or so. 

And @alebcay is correct, you need a new line.

**Comment by mzgnr** [comment]
@joshtoo thanks for the tip.

**Comment by shockone** [comment]
@mzgnr, sorry, I haven't noticed your PR and added the same changes.

**Pull request closed without merging**